sbt-plugin/src/main/scala/org/scalajs/sbtplugin/LinkerImpl.scala

```diff
@@ -75,6 +75,7 @@ object LinkerImpl {
     private val parentPrefixes = List(
       "java.",
       "scala.",
+      "sun.",
```

Comment:
We'll need a comment explaining why we have this, with a reference to the issue and possibly also the OpenJDK issue.

Reply:
ok. I'll add